app/src/main/res/layout/item_status_pr.xml

```diff
@@ -1,54 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical" android:layout_width="match_parent"
+    xmlns:tool="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_margin="5dp"
+    android:layout_margin="@dimen/margin_top"
     android:background="@android:color/white"
-    android:padding="15dp"
+    android:padding="@dimen/card_header_margin"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    app:elevation="2dp"
-  >
+    app:elevation="@dimen/card_elevation">
 
     <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
         android:orientation="vertical" android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_margin="10dp"
+        android:layout_margin="@dimen/card_margin"
         android:background="@android:color/white">
 
 
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/tv_pr_number"
-            android:text="#319"
-            android:textSize="20sp"
+            tool:text="#319"
+            android:textSize="@dimen/text_heading"
             android:textColor="@color/colorAccent"/>
 
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/tv_pr_repo"
-            android:text="openMF/community-app"
-            android:textSize="16sp"
-            android:paddingTop="5dp"
+            tool:text="openMF/community-app"
+            android:textSize="@dimen/text_normal"
+            android:paddingTop="@dimen/margin_top"
             android:textColor="@android:color/black"/>
 
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:id="@+id/tv_pr_title"
-            android:textSize="16sp"
-            android:paddingTop="10dp"
-            android:text="Fix issue 217 and other imrpvements" />
+            android:textSize="@dimen/text_normal"
+            android:paddingTop="@dimen/card_margin"
+            tool:text="Fix issue 217 and other imrpvements" />
```

Comment:
Spelling mistake

Reply:
oh, I just forget to correct spelling as for this PR I only focused on removing the hardcoded values.